h2o-core/src/test/java/water/parser/ParseTimeTest.java

```diff
@@ -293,4 +293,27 @@
       Assert.assertEquals(exp[i],vec.at8(i));
     fr.delete();
   }
+
+  @Test public void testISO8601_nodashes() {
+    // ISO 8601 allows YYYY-MM-DD as well as YYYYMMDD
+    String data = "Date\n"+
```

Comment:
What about negative cases? `14920123-25:00:78`? Can we have a small test to expected verify `Long.MIN_VALUE` .